Cargo.toml

```diff
@@ -2,12 +2,18 @@
 name = "navigator-rs"
 version = "0.1.0"
 edition = "2021"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+license = "MIT"
+description = "This crate serves as the entry point for embedding applications using Rust on Blue Robotics’s Navigator"
+homepage = "https://bluerobotics.com/store/comm-control-power/control/navigator/"
+documentation = "https://docs.bluerobotics.com/navigator-rs/navigator_rs/"
+repository = "https://github.com/bluerobotics/navigator-rs"
+categories = ["BlueRobotics", "Embedded development", "Science"]
+keywords = ["BlueRobotics", "embedded", "ROV", "robotics", "SDK"]
```

Comment:
Add navigator and the sensors name as well.

Reply:
keywords are limited to 5.

Reply:
Remove sdk and add navigator

Reply:
it's done